### PR TITLE
Fix link to Style docs following nbsphinx move

### PR DESCRIPTION
**Pull request by jnothman**

### CODE REVIEW

**Comment by jreback** [comment]
thanks!

File: doc/source/whatsnew/v0.20.0.txt [hunk 1]
```diff
@@ -430,7 +430,7 @@ For example, after running the following, ``styled.xlsx`` renders as below:
     import os
     os.remove('styled.xlsx')
 
-See the :ref:`Style documentation <style>` for more detail.
+See the :ref:`Style documentation <style.ipynb>` for more detail.
```
Comment (jreback) [inline doc/source/whatsnew/v0.20.0.txt]: actually would be better for this one to go directly to: http://pandas-docs.github.io/pandas-docs-travis/style.html#Export-to-Excel

Comment (jnothman) [inline doc/source/whatsnew/v0.20.0.txt]: True. Are you expecting another PR?

Comment (jreback) [inline doc/source/whatsnew/v0.20.0.txt]: if you can would be great.

**Pull request merged**